### PR TITLE
Add 'excludeLabels' to exclude issues matching a label

### PR DESCRIPTION
**Pull request by zarenner**
e.g. to be able to query non-reliability bugs

### CODE REVIEW

**Comment by zarenner** [comment]
example report: https://github.com/github/c2c-actions-platform/blob/zarenner/update-issues-report/IssuesReport.md#-Reliability-bugs-query

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -31,3 +31,8 @@ jobs:
         outputPath: 'IssuesReport.md'
         configPath: 'test/issues-report.json'
         token: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/upload-artifact@v2
+      with:
+        # This file gets produced by main.test.ts
```
Comment (brcrista) [inline .github/workflows/test.yml]: For this job, `IssuesReport.md` will be produced by the action itself, right?

File: src/markdown.ts [hunk 2]
```diff
@@ -54,22 +54,21 @@ const h2 = (text: string) => `## ${text}`;
...
 const code = (text: string) => `\`${text}\``;
+const strikeCode = (text: string) => `\~\`${text}\`\~`;
 
 // Useful for converting a header name to an HTML ID in a hacky way
 const hyphenate = (headerName: string) => headerName.replace(/\s+/g, '-');
 
 /** Construct a URL like `https://github.com/brcrista/summarize-issues-test/issues?q=is%3Aissue+is%3Aopen+label%3Aincident-repair+label%3Ashort-term`. */
-function issuesQuery(repoContext: RepoContext, labels: string[], assignee?: string) {
+function issuesQuery(repoContext: RepoContext, labels: string[], excludeLabels: string[], assignee?: string) {
     // If the label contains a space, the query string needs to have it in quotes.
```
Comment (brcrista) [inline src/markdown.ts]: Could you move this comment down to live with `makeLabelsUrlSafe` function?

File: dist/index.js [hunk 3]
```diff
@@ -8922,20 +8926,17 @@ const h2 = (text) => `## ${text}`;
 const h3 = (text) => `### ${text}`;
 const link = (text, href) => `[${text}](${href})`;
 const code = (text) => `\`${text}\``;
+const strikeCode = (text) => `\~\`${text}\`\~`;
```
Comment (brcrista) [inline dist/index.js]: I like this way of representing excluded labels 😆
To match the other helpers, you could decompose into

```suggestion
const strike = (text) => `\~${text}\~`;
```

and then call it with

```js
.map(x => strike(code(x)))
```

**Review by brcrista — APPROVED** [review]
Thank you!  🙇‍♂️

File: .github/workflows/test.yml (hunk 1 above)
Comment (zarenner) [reply .github/workflows/test.yml]: yes, I'll update the comment, thanks :-)

**Comment by zarenner** [comment]
@brcrista made the updates, mind merging if it looks good?

**Pull request merged**